README.md

```diff
@@ -1,17 +1,18 @@
 # Continous Deployment
 
-This repository contains an opinionated reference architecture to setup, manage and operate a continous deployment pipeline.
+[This repository](https://github.com/operate-first/continuous-deployment) contains an opinionated reference architecture to setup, manage and operate a continous deployment pipeline.
 
 ### Prerequisites
-Kustomize 3.8.1+
-SOPS 3.4.0+
-KSOPS 2.1.2+
+* Kustomize 3.8.1+
+* SOPS 3.4.0+
+* KSOPS 2.1.2+
+Find more informaton on how to install [here](docs/manage_your_app_secrets.md).
 
-Ensure you have the key to decrypt secrets. Reach out to members of the Data Hub team for access.
+Ensure you have the key to decrypt secrets. Reach out to members of the Data Hub team(data-hub-users@redhat.com) for access.
```

Comment:
I get the feeling this email is wrong.

Reply:
do you know the right one? is it perhaps, data-hub@redhat.com?

I think I'm subscribed to this one where the users ask questions
![image](https://user-images.githubusercontent.com/32435206/101045647-70231280-354e-11eb-88bc-d44a4c9016b8.png)

Reply:
Well, we generally  ask folks to email `data-hub@redhat.com`. That being said, these docs are for general operate-first right? data-hub team is the wrong one to refer to here I think. @HumairAK am i mistaken?

Reply:
So should we completely remove that then? Probably ask them to open an issue in this repo?

Reply:
I think changing it to say "open an issue in this repo" is the right call

Reply:
Done